en-US/privacy/index.lang

```diff
@@ -15,6 +16,11 @@ Contact Mozilla
 If you want to make a correction to your information, or you have any questions about our privacy policies, please get in touch with:
 
 
+## TAG: privacy_email_2019
+;See <a href="https://app.onetrust.com/app/#/webform/4ba08202-2ede-4934-a89e-f0b0870f95f0">here</a> for Data Subject Access Requests.
+See <a href="https://app.onetrust.com/app/#/webform/4ba08202-2ede-4934-a89e-f0b0870f95f0">here</a> for Data Subject Access Requests.
```

Comment:
Why is this URL exposed to localization, instead of being in the code? It really shouldn't, and will require to invalidate the string if it ever gets updated. It should be a variable instead.